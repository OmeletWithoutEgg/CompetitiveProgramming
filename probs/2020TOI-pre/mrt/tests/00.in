9
2 1 1 0 1 1 1
7 4 4 0 2 5 3
23 12 10 21 1 20 4
5 1 4 2 3 3 2
19 3 17 15 6 5 10
23 19 10 15 6 22 6
5 2 4 4 4 2 1
7 6 0 5 6 4 2
3 2 0 2 0 0 0
