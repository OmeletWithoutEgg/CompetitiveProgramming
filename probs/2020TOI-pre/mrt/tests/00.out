1 1
4 0
14 22
1 4
6 2
9 6
0 3
0 0
1 0
